src/main/scala/io/citrine/lolo/stats/utils.scala

```diff
@@ -0,0 +1,70 @@
+package io.citrine.lolo.stats
+
+import io.citrine.lolo.linear.LinearRegressionLearner
+
+import scala.util.Random
+
+object utils {
```

Comment:
This is purely stylistic. But it seems like the library convention is UpperCamelCase for all objects and classes.

Should this be renamed to `StatsUtils` ?